.github/workflows/master.yml

```diff
@@ -23,6 +23,10 @@ jobs:
     - name: Lint code
       run: npm run lint
 
+    - name: Prepare config
+      run: |
+        cp packages/deployer/src/config-data/config.json.example packages/deployer/src/config-data/config.json
```

Comment:
Even though the node package no longer needs `config.json` to build, the deployer still does.